.travis.yml

```diff
@@ -2,21 +2,16 @@ language: node_js
 sudo: false
 dist: precise
 node_js:
+    - "node"
```

Comment:
This specifies simply the latest stable version of Node, which in this case is v8.0.0. I could add 8 explicitly and leave this in case the file isn't updated when a release comes out?